examples/data_pipeline/pytext_vocab.py

```diff
@@ -0,0 +1,183 @@
+# from collections import OrderedDict
+
+# from fairseq.data.dictionary import Dictionary
+import torch
+from torchtext.experimental.vocab import Vocab
+from typing import List, Optional
+
+
+# TODO: This function is commented out since we don't have fairseq as a dependency in torchtext.
```

Comment:
The dependencies only really apply at a package level, it's ok to add it for the examples.

Reply:
And should we uncomment `build_fairseq_vocab` func below as an example for pytext to load fairseq vocab?

Reply:
Done